torch_frame/data/multi_embedding_tensor.py

```diff
@@ -117,6 +94,64 @@ def from_tensor_list(
         offset = torch.LongTensor(offset_list)
         return cls(num_rows, num_cols, values, offset)
 
+    def __getitem__(
+        self,
+        index: Any,
+    ) -> Union["MultiEmbeddingTensor", Tensor]:
+        if isinstance(index, tuple) and len(index) == 2 and isinstance(
+                index[0], int) and isinstance(index[1], int):
+            i = self._normalize_index(index[0], dim=0)
+            j = self._normalize_index(index[1], dim=1)
+            return self.values[i, self.offset[j]:self.offset[j + 1]]
+        if isinstance(index, int):
+            index = self._normalize_index(index, dim=0)
+            return MultiEmbeddingTensor(
+                num_rows=1,
+                num_cols=self.num_cols,
+                values=self.values[index].view(1, -1),
+                offset=self.offset,
+            )
+        if isinstance(index, Tensor) and index.ndim == 1:
```

Comment:
I think we need to check the device here and change the device of the index if it's different from `MultiEmbeddingTensor`. Do you mind also adding gpu testing? Thanks

Reply:
Just to be clear, by "GPU testing", we do not mean adding a new GPU unit test. Just running an adhoc test on your local GPU machine is enough for now.

Reply:
To make the behaviour consistent with PyTorch which accepts both cpu/cuda tensors as index, I'll just keep this as is, but I'll enable gpu testing for this test case.